src/components/MultiHopTrade/components/VerifyAddresses/VerifyAddresses.tsx

```diff
@@ -87,15 +90,14 @@ export const VerifyAddresses = () => {
     [isAddressVerified, sellAddress],
   )
   const buyVerified = useMemo(
-    () => isAddressVerified(buyAddress ?? ''),
-    [buyAddress, isAddressVerified],
+    () => maybeManualReceiveAddress || isAddressVerified(buyAddress ?? ''),
```

Comment:
nitpick refine type (boolean your fav! haha)

```suggestion
    () => Boolean(maybeManualReceiveAddress) || isAddressVerified(buyAddress ?? ''),
```

Reply:
I like Boolean